### PR TITLE
Island name

### PR DESCRIPTION
**Pull request by PeachesMLG**
Fixes https://github.com/Iridium-Development/IridiumSkyblock/issues/362 by allowing the island name to be stored as null in the database and then defaulting it to the owners name.

Only issue is how to transform this data in sqllite since ALTER apparently doesnt exist?

### CODE REVIEW

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 1]
```diff
@@ -66,7 +66,10 @@ public boolean execute(CommandSender sender, String[] args) {
             ));
             return false;
         }
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:68]:
```suggestion
        }
        
```

File: src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java [hunk 2]
```diff
@@ -66,7 +66,10 @@ public boolean execute(CommandSender sender, String[] args) {
...
-
+        if (IridiumSkyblock.getInstance().getIslandManager().getIslandByName(name).isPresent()) {
+            player.sendMessage(IridiumSkyblock.getInstance().getMessages().islandWithNameAlreadyExists.replace("%prefix%", (IridiumSkyblock.getInstance().getConfiguration()).prefix));
+            return false;
+        }
```
Comment (dlsf) [inline src/main/java/com/iridium/iridiumskyblock/commands/RenameCommand.java:72]:
```suggestion
        }
        
```

**Review by dlsf — CHANGES_REQUESTED** [review]

File: src/main/resources/patch_2_mysql.sql [hunk 3]
```diff
@@ -0,0 +1 @@
+ALTER TABLE islands MODIFY name VARCHAR(512) NOT NULL
```
Comment (PeachesMLG) [inline src/main/resources/patch_2_mysql.sql:1]: Shouldnt this be null?

Comment (dlsf) [reply src/main/resources/patch_2_mysql.sql:1]: maybe

**Pull request merged**